flight/Modules/RadioComBridge/radiocombridge.c

```diff
@@ -69,7 +69,9 @@
 #define TASK_PRIORITY     PIOS_THREAD_PRIO_LOW
```

Comment:
The content here doesn't match the commit message? (although that is here too)

Reply:
Aw shit.  took me awhile to figure out what you meant.. will try to fix.

Reply:
OK, forcepushed.  thanks for noticing, I have problems with sometimes flattening commits together during rebases.